Mailer/Mailer.php

```diff
@@ -91,16 +92,40 @@ public function sendResettingEmailMessage(UserInterface $user)
      */
     protected function sendEmailMessage($renderedTemplate, $fromEmail, $toEmail)
     {
+        if (null === $this->mailer) {
+            throw new \RuntimeException(
+                'Sending email requires the "mailer" service to be available. '.
+                'Run "composer require symfony/mailer" or "composer require symfony/swiftmailer-bundle"'
+            );
+        }
+
+        if (!in_array(get_class($this->mailer), ['Symfony\Component\Mailer\MailerInterface', 'Swift_Mailer'], true)) {
+            throw new \RuntimeException(
+                'Sending email requires either symfony/mailer or symfony/swiftmailer-bundle'.
+                'Run "composer require symfony/mailer" or "composer require symfony/swiftmailer-bundle"'
+            );
+        }
+
         // Render the email, use the first line as the subject, and the rest as the body
         $renderedLines = explode("\n", trim($renderedTemplate));
         $subject = array_shift($renderedLines);
         $body = implode("\n", $renderedLines);
 
-        $message = (new \Swift_Message())
-            ->setSubject($subject)
-            ->setFrom($fromEmail)
-            ->setTo($toEmail)
-            ->setBody($body);
+        if (class_exists('Symfony\Component\Mailer\MailerInterface')) {
+            $message = (new Symfony\Component\Mime\Email())
+                ->subject($subject)
+                ->from($fromEmail)
+                ->to($toEmail)
+                ->html($body);
+        }
+
+        if (class_exists('Swift_Mailer')) {
```

Comment:
This seems dangerous - yes you cant have swiftmailer-bundle AND symonfy/mailer - but you can have symfony/mailer AND swiftmailer (without the bundle.)